ateam_ai/src/behavior/behavior_executor.cpp

```diff
@@ -50,15 +48,25 @@ void BehaviorExecutor::execute_behaviors(
   //
 
   // Send commands down to motion control
-  Eigen::Vector2d target = std::get<MoveParam>(
-    behaviors.get_node(
-      behaviors.get_root_nodes().front()).params).target_location;
-  Eigen::Vector2d command = (target - world.our_robots.at(0).value_or(Robot()).pos) / 1000.0;
-  if (command.norm() > 1) {
-    command = command.normalized();
+  RobotMotionCommands robot_motion_commands;
+  for (std::size_t id = 0; id < robot_motion_commands.size(); id++) {
+    // Note: This abuses the way the behaviors are supposed to be executed
+    // This is just a placeholder for now
+    Eigen::Vector2d target = std::get<MoveParam>(
+      behaviors.get_node(
+        behaviors.get_root_nodes().front()).params).target_location;
+
+    Eigen::Vector2d command = (target - world.our_robots.at(id).value_or(Robot()).pos) / 1000.0;
+
+    if (command.norm() > 1) {
+      command = command.normalized();
```

Comment:
There's a function for normalizing the vector in place.

```suggestion
      command.normalize();
```